core/new-gui/src/app/dashboard/user/service/workflow-version/workflow-version.service.ts

```diff
@@ -36,28 +37,29 @@ const ID_FILED_FOR_ELEMENTS_CONFIG: { [key: string]: string } = {
 export class WorkflowVersionService {
   public modificationEnabledBeforeTempWorkflow = true;
   public operatorPropertyDiff: { [key: string]: Map<String, String> } = {};
-  private workflowVersionsObservable = new Subject<readonly string[]>();
   private displayParticularWorkflowVersion = new BehaviorSubject<boolean>(false);
   private differentOpIDsList: DifferentOpIDsList = { modified: [], added: [], deleted: [] };
 
+  private workflowVersionsSubject = new ReplaySubject<string>(1);
```

Comment:
why use replay subject?

Reply:
I want this service to keep a "current state"; replay by 1  message can always give the new subscribers update about the current state.